### ISSUE TITLE
RedPanda as Kafka alternative?

### ISSUE THREAD

**Issue by chrisgoddard** [TrivadisPF/platys-modern-data-platform#622]
Any thoughts on adding RedPanda as Kafka alternative - I've generally found it performs better (at least for some applications) and doesn't require zookeeper

**Comment by gschmutz** [comment]
@chrisgoddard yes I have thought about, just to offer it as an option. I'm not really targeting production use cases with this generator, so performance is of less importance to be but to offer it as an option for PoC why not? I have actually added an issue #403 for it a while ago .... put it didn't get the priority it might deserve ;-) 

On the Zookeeper point, with Kafka 3.3 and Confluent 7.3.0 Zookeeper-less mode is GA. I support it but haven't tested for a long time, will also write a cookbook recipe documenting it.
